scripts/run-demo.sh

```diff
@@ -5,8 +5,8 @@ set -e
 genesis_time=$(date -d "$(mina advanced compile-time-constants | jq -r '.genesis_state_timestamp')" +%s)
 now_time=$(date +%s)
 
-export CODA_TIME_OFFSET=$(( $now_time - $genesis_time ))
+export CODA_TIME_OFFSET=0
```

Comment:
I thought we need this to start from genesis block even on a build that sets genesis time long in the past

Reply:
these builds set genesis time to like, yesterday, so the complexity here shouldn't be needed, and it was producing a negative value and therefore never producing blocks when figment was testing with it.